cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
Previously I've tried setting it in `hunter.cmake` with `hunter_cmake_args ` as per your suggestion, but I felt that was rather fragile since I could not specify the version that those parameters would apply. 

If one would chose to use a previous version that was not patched for the Qt backend, such as 2.4 or 3.0, the build would break.